ReactiveCocoaTests/Swift/TestError.swift

```diff
@@ -14,3 +17,27 @@ enum TestError: Int {
 
 extension TestError: ErrorType {
 }
+
+
+internal extension SignalProducerType {
+	/// Halts if an error is emitted in the receiver signal.
+	/// This is useful in tests to be able to just use `startWithNext`
+	/// in cases where we know that an error won't be emitted.
+	func assumeNoErrors() -> SignalProducer<Value, NoError> {
+		return self.lift { $0.assumeNoErrors() }
+	}
+}
+
+internal extension SignalType {
+	/// Halts if an error is emitted in the receiver signal.
+	/// This is useful in tests to be able to just use `startWithNext`
+	/// in cases where we know that an error won't be emitted.
+	func assumeNoErrors() -> Signal<Value, NoError> {
```

Comment:
This doesn't _suppress_ them though: I feel strongly we shouldn't expose such operator. This **assumes** that none will be emitted, resulting in a runtime failure if they are.